package.json

```diff
@@ -3,7 +3,8 @@
     "version": "1.0.0",
     "lastUpdate": {
         "vellore": "Winter Semester 2022 (Freshers)",
-        "chennai": "Winter Semester 2023-24"
+        "chennai": "Winter Semester 2023-24",
+        "ap": "Fall Semester 2023-24"
```

Comment:
You mean?
```suggestion
        "ap": "Fall Semester 2024-25"
```

Reply:
Our Fall semester for 2024-25 has not yet started. The data currently available pertains to the Fall semester of 2023-24. Since the campus is new, the teachers and venues change every semester, which means we do not have fixed data available. I have gathered this information from various friends and teachers and circulated an Excel sheet to compile what I could. I will make an effort to obtain the latest data for the Fall semester 2024-25 once the registration period begins, which will be in a month or later.